topo/node/cisco/cisco.go

```diff
@@ -413,6 +438,13 @@ func defaults(pb *tpb.Node) (*tpb.Node, error) {
 		if pb.Config.Image == "" {
 			pb.Config.Image = "8000e:latest"
 		}
+		if len(pb.HostConstraints) == 0 {
```

Comment:
this condition will still never happen because of line 427

Reply:
line 427 is for model xrd. This line checks for hostconstraints in 8000e series based on a switch case. So this condition would happen, right?

Reply:
Resolved offline.